charts/airbyte-server/values.yaml

```diff
@@ -94,8 +94,8 @@ containerSecurityContext: {}
 livenessProbe:
   enabled: true
   initialDelaySeconds: 30
-  periodSeconds: 10
-  timeoutSeconds: 1
+  periodSeconds: 30
+  timeoutSeconds: 30
```

Comment:
do we actually want to change the liveness and readiness probes to 30 second period and timeouts?  Wouldn't changing it here would change the behavior of the deployment outside of just tests?

Reply:
We do, since server takes some time to startup we need to increase the timeouts and request time so the request wont fail instantly on getting headers or getting refused at all, which lead to server pod being recreated indefinitely